src/app/pages/ServiceRegistry/ServiceRegistryPage.tsx

```diff
@@ -19,7 +20,7 @@ export const ServiceRegistryPage: React.FunctionComponent = () => {
 export const ServiceRegistryPageConnected: React.FC = () => {
   const config = useConfig();
   const { getQuota } = useQuota(ProductType.srs);
-  const { preCreateInstance, shouldOpenCreateModal } = useModalControl();
+  const { preCreateInstance, shouldOpenCreateModal } = useModalControl(termsConfig['service-registry'].EventCode);
```

Comment:
Can you add a `type` for the `termsConfig`?

Reply:
done